CDCgov/cdc-open-viz
# fix: dev-9142 Fix missing first grid line

This fixes a bug where the first grid line sometimes isn't shown. It happens often with the respiratory virus charts that are [currently live](https://www.cdc.gov/respiratory-viruses/data/activity-levels.html). It would be great if this could go out in the September release.

The bug was introduced in [this commit](https://github.com/CDCgov/cdc-open-viz/pull/1432/commits/8ba391bf64bce964196b9df397bbdf5fd599a864) which was related to DEV-7953.

## Testing Steps

The `testposorig.json` file attatched to the Jira ticket can be used to see the issue this fixes. Load that file on the `dev` branch, see that it's broken, then see that this branch fixes it.
